pkg/util/util_test.go

```diff
@@ -70,3 +71,51 @@ func TestGetSliceIntersection(t *testing.T) {
 		})
 	}
 }
+
+func TestChildPath(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name   string
+		base   string
+		target string
+		exp    string
+		err    string
+	}{
+		{
+			name:   "success",
+			base:   ".",
+			target: "./terraform/project",
+			exp:    "terraform/project",
+		},
+		{
+			name:   "empty",
+			base:   "",
+			target: "",
+			exp:    "",
+		},
+		{
+			name:   "not_child_dir",
+			base:   "./terraform/project",
+			target: ".",
+			err:    "is not a child of",
+		},
```

Comment:
Can you add some more exhaustive test cases here, including things like filepaths with spaces and special characters?